docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Device_operations/Upgrading_firmware.md

```diff
@@ -1,10 +1,10 @@
-# Upgrading device firmware
+# Upgrade device firmware
 
 If you would like to upgrade the firmware of devices using the Azure IoT Hub, follow the instruction below.
 
 ## Prerequisites
 
- - At least one device with active Coiote DM - Azure IoT Hub [synchronization](../Importing_devices_to_Coiote_DM).
+ - At least one device with active Coiote DM - Azure IoT Hub [synchronization](../Synchronize_devices_with_Azure_IoT_Hub).
```

Comment:
maybe `with the active Coiote DM - Azure IoT Hub integration` ?

Reply:
yes indeed